.config-schema.json

```diff
@@ -56,6 +56,16 @@
             "default": 1000,
             "x-env-variable": "OPENFGA_LIST_OBJECTS_MAX_RESULTS"
         },
+        "latencyDBQueryCountBuckets": {
```

Comment:
To be consistent with vernacular elsewhere, and the label itself, please use `Datastore` instead of `DB`.